### PR TITLE
use string-concatenation instead of sprintf for simple cases

### PR DESCRIPTION
**Pull request by thaJeztah**
Signed-off-by: Sebastiaan van Stijn <github@gone.nl>

### CODE REVIEW

File: libcontainer/apparmor/apparmor.go [hunk 1]
```diff
@@ -35,14 +33,13 @@ func setProcAttr(attr, value string) error {
 		return err
 	}
 
-	_, err = fmt.Fprintf(f, "%s", value)
+	_, err = fmt.Fprint(f, value)
```
Comment (kolyshkin) [inline libcontainer/apparmor/apparmor.go:36]: It's better to use
```go
_, err := `f.WriteString(value)`
```
instead I guess.

File: libcontainer/cgroups/systemd/common.go [hunk 2]
```diff
@@ -307,7 +307,7 @@ func newProp(name string, units interface{}) systemdDbus.Property {
 func getUnitName(c *configs.Cgroup) string {
 	// by default, we create a scope unless the user explicitly asks for a slice.
 	if !strings.HasSuffix(c.Name, ".slice") {
-		return fmt.Sprintf("%s-%s.scope", c.ScopePrefix, c.Name)
+		return c.ScopePrefix + "-" + c.Name
```
Comment (kolyshkin) [inline libcontainer/cgroups/systemd/common.go:310]: You lost the `.scope` part; I guess CI failures are because if this

File: utils_linux.go [hunk 3]
```diff
@@ -214,13 +214,13 @@ func createPidFile(path string, process *libcontainer.Process) error {
...
 		tmpDir  = filepath.Dir(path)
-		tmpName = filepath.Join(tmpDir, fmt.Sprintf(".%s", filepath.Base(path)))
+		tmpName = filepath.Join(tmpDir, "."+filepath.Base(path))
 	)
 	f, err := os.OpenFile(tmpName, os.O_RDWR|os.O_CREATE|os.O_EXCL|os.O_SYNC, 0666)
 	if err != nil {
 		return err
 	}
-	_, err = fmt.Fprintf(f, "%d", pid)
+	_, err = fmt.Fprint(f, strconv.Itoa(pid))
```
Comment (kolyshkin) [inline utils_linux.go:223]: `f.WriteString`

File: libcontainer/apparmor/apparmor.go (hunk 1 above)
Comment (thaJeztah) [reply libcontainer/apparmor/apparmor.go:36]: Ah, yes, didn't think of that 👍, will update

File: libcontainer/cgroups/systemd/common.go (hunk 2 above)
Comment (thaJeztah) [reply libcontainer/cgroups/systemd/common.go:310]: 🤦 thanks; good that we have reviews and CI

**Comment by thaJeztah** [comment]
Updated; PTAL 🤗

File: libcontainer/system/proc.go [hunk 4]
```diff
@@ -78,7 +78,7 @@ func GetProcessStartTime(pid int) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return fmt.Sprintf("%d", stat.StartTime), nil
+	return strconv.FormatUint(stat.StartTime, 10), nil
```
Comment (kolyshkin) [inline libcontainer/system/proc.go:81]: Wonder if `GetProcessStartTime` has any users left? I looked around, can't find any. Maybe we can just remove it.

Comment (kolyshkin) [reply libcontainer/system/proc.go:81]: Can and should be addressed separately of course.

**Review by kolyshkin — APPROVED** [review]
LGTM

**Review by AkihiroSuda — APPROVED** [review]

**Pull request merged**